lib/util/macros/src/lib.rs

```diff
@@ -8,8 +8,8 @@ use syn::parse::{Parse, ParseStream};
 use syn::token::Return;
 use syn::{braced, bracketed, Expr, Ident, Result, Token};
 
-// 1 min
-const DEFAULT_TIMEOUT: u64 = 45 * 1000;
+// In seconds
+const DEFAULT_TIMEOUT: u64 = 40 * 1000;
```

Comment:
I commented on this downstack, I think it's probably better to add changes to that file just so I'm not looking at outdated things there first :) might be something to figure out for our Graphite team cohesion down the line :+